Gruntfile.js

```diff
@@ -96,7 +96,9 @@ grunt.initConfig({
 			"test/autostart.html",
 			"test/startError.html",
 			"test/logs.html",
-			"test/setTimeout.html"
+			"test/setTimeout.html",
+			"test/reporter-html-legacy-markup.html",
+			"test/reporter-html-no-qunit-element.html"
```

Comment:
Without the reporter fix below, this test just times out. Let's try to improve this to produce a useful failure.